assets/js/googlesitekit/datastore/site/connection.js

```diff
@@ -113,6 +137,23 @@ export const resolvers = {
 };
 
 export const selectors = {
+	/**
+	 * Gets the connection info for this site.
+	 *
+	 * Returns `null` if the connection info is not available/loaded.
+	 *
+	 * Returns an object with the shape when successful:
+	 * ```
+	 * {
+	 *   connected: <Boolean>,
+	 *   resettable: <Boolean,
+	 * }
+	 * ```
+	 *
+	 * @since @n.e.x.t
+	 * @param {Object} state Data store's state.
+	 * @return {Object|null} Site connection info.
```

Comment:
Nitpick but JSdoc says multiple types are supposed to be surrounded by parenthesis. I prefer without myself but we should follow this https://jsdoc.app/tags-type.html